packages/integrations/cloudflare/README.md

```diff
@@ -38,50 +38,47 @@ export default defineConfig({
 
 ## Options
 
-### Mode
-
-`mode: "advanced" | "directory"`
+### `mode: "advanced" | "directory"`
```

Comment:
Thank you for the insights; I hadn't noticed the visual aspect. I've chosen this code format for now.

Is there a docs preview available for PRs so I can review the changes visually?

Reply:
Sadly no! It's on our wishlist, and we believe it's possible (since a build of Docs needs to be run in order to make the automated PR to the docs repo) but we haven't had the time to explore hooking into that for a preview in these PRs.